src/main/java/edu/kis/vh/nursery/list/IntArrayStack.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery.list;
+
+public class IntArrayStack implements ListInterface {
+    public IntArrayStack(int size) {
+        IntArrayStack.size = size ;
+    }
+    public static int size=12;
```

Comment:
size shouldn't be a static variable (changing for one instance, changes it for another). It was meant to be a constant (static final)